metaTransactions/biconomy.md

```diff
@@ -0,0 +1,99 @@
+
+Biconomy supports Gnosis contract wallet integration. Checkout how you can integrate meta transactions via Gnosis smart contract wallet here.
+
+### 2. Custom Implementation
+
+If dApps support native meta transactions, then Biconomy's relayers would directly relay the transactions to the network without the need for any proxy contract.
+
+![../images/biconomy/NativeMetaTx.png](../images/biconomy/NativeMetaTx.png)
+
+### 3. EIP 2771 Standard Implementation
+
+Instead of integrating meta transaction validation logic directly into your contract - you can inherit a recipient contract that can accept validated calls from a trusted forwarder. The trusted forwarder complies with EIP 2771 and verifies signatures before calling smart contract with the original user address data appended. It makes development easier and gives you the flexibility to change the trusted forwarder address when your needs change.
+
+![../images/biconomy/trustedForwarder.png](../images/biconomy/trustedForwarder.png)
+
+Using Biconomy's Mexa SDK you can integrate Biconomy into your DApp seamlessly. Using Biconomy's dashboard gives you a fine control on settings and configuration for enabling MetaTransactions.
+
+## Integration
+
+**Integration using Mexa is a simple step process:**
+
+1. Register your DApp on Biconomy Dashboard, a dashboard for developers, and copy API Key generated for your DApp.
+2. Add the contract address and contract ABI on which you wanna enable Meta Transaction to your Biconomy dashboard.
+3. Integrate Mexa SDK in your DApp code using API Key you got from dashboard.
+
+### Dashboard
+
+Follow the steps **[here](https://docs.biconomy.io/biconomy-dashboard)**, to register an account and add a DApp to get the keys, and configure functions that will accept signed transactions.
+
+### Using mexa
+
+1. Get inside the DApp client code directory, to configure meta transactions. Let's first install `@biconomy/mexa` from npm.
+
+   ```jsx
+   npm install @biconomy/mexa --save
+   ```
+
+2. Now you need to initialize Biconomy & web3. In place of `<web3 provider>` you can use `window.ethereum` if your DApp users are using MetaMask.
+
+```jsx
+import { Biconomy } from "@biconomy/mexa";
+
+const biconomy = new Biconomy(<web3 provider>, {apiKey: <API Key>});
+web3 = new Web3(biconomy);
+
+biconomy.onEvent(biconomy.READY, () => {
+
+  // Initialize your dapp here like getting user accounts, initialising contracts and etc
+
+}).onEvent(biconomy.ERROR, (error, message) => {
+
+  // Handle error while initializing mexa
+
+});
+```
+
+## 🥳
+
+## Congrats, you've successfully integrated Biconomy into your Dapp and now your dapp supports meta Tx.
+
+### Wanna make your users pay gas-fee in ERC20 tokens? We can help you with that. Read more about that on our [docs](http://docs.biconomy.io).
+
+**Wanna get in touch?**
```

Comment:
you might want to turn this into a header so that it appears on the table of contents, then users will be able to quickly get to your contact information... but ultimately it's up to you! 😄